automatic/vlc.portable/README.md

```diff
@@ -0,0 +1,27 @@
+# <img src="https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@3952380efd0776bea6d964ed4e28efa979b434d2/icons/vlc.png" width="48" height="48"/> [vlc](https://chocolatey.org/packages/vlc)
+
+
+VLC is a free and open source cross-platform multimedia player and framework that plays most multimedia files as well as DVD, Audio CD, VCD, and various streaming protocols.
+
+![Screenshot](https://i.imgur.com/DEG19Fs.png)
```

Comment:
Screenshot should be in folder of virtual package, lets not depend on random site.

Reply:
Done. The link will be invalid till this is merged.